### ISSUE TITLE
No product reader compatible with /data/PRODUCTS/Landsat8/196/30

### ISSUE THREAD

**Issue by paugalles** [senbox-org/sen2like#11]
From where should I download landsat products in a format that is compatible with sen2like tool? I have tried downloading from [https://earthexplorer.usgs.gov/](https://earthexplorer.usgs.gov/) and I get the following error in the processor:

`No product reader compatible with /data/PRODUCTS/Landsat8/196/30`

The command that I ran:

`python sen2like/sen2like/sen2like.py product-mode /data/PRODUCTS/Landsat8/196/30 --tile 31TFJ --wd /data/wd --conf sen2like/conf/config.ini --no-run`

**Comment by yorik31** [comment]
Sen2Like did not recognize  "/data/PRODUCTS/Landsat8/196/30" as a product, it makes sense.

if update your previous command line, with  product path, this should work.

As shown  in the user manual 
https://github.com/senbox-org/sen2like/blob/master/sen2like/docs/source/S2-SEN2LIKE-UM-V1.9.pdf
( pp  34 $4.3.3)

We hope it helps !!!!

**Comment by paugalles** [comment]
@yorik31 , thanks for your prompt response
Does the product indicated as an argument has to be always a Sentinel2 product?

**Comment by yorik31** [comment]
No, it works also with Landsat 8/9 C1/C2 Level 1 products, and PRISMA Level 1 products. 
All of this information is captured within the UM document.
